frontend/src/js/components/IngestionEvents/IngestionEvents.tsx

```diff
@@ -158,11 +128,12 @@ const parseBlobStatus = (status: any): BlobStatus => {
 }
 
 export function IngestionEvents(
-    {collectionId, ingestId, workspaces, breakdownByWorkspace}: {
+    {collectionId, ingestId, workspaces, breakdownByWorkspace, showErrorsOnly}: {
         collectionId: string,
         ingestId?: string,
         workspaces: WorkspaceMetadata[],
-        breakdownByWorkspace: boolean
+        breakdownByWorkspace: boolean,
+        showErrorsOnly?: boolean,
```

Comment:
It looks like you could make this required rather than optional - as far as I can tell it is always defined